client/src/stores/modules/cart.js

```diff
@@ -16,8 +16,18 @@ export default {
   },
   mutations: {
     addToCart(state, item){
-        state.cart.push(item);
-        saveCart(state.cart);
+      state.cart.push(item);
+      saveCart(state.cart);
+    },
+    removeFromCart(state, item){
+      let itemIndex = state.cart.indexOf(item);
+      state.cart.splice(itemIndex, 1);
+      saveCart(state.cart);
+    }
+  },
+  actions: {
+    persistCart(context) {
+      saveCart(context.state.cart);
```

Comment:
The store is currently the only thing that has the ability to persist the cart to local storage. So I've added the action to be able to trigger that when necessary. Right now that only needs to be triggered externally from the store when quantity values are changed.